gradle/libs.versions.toml

```diff
@@ -47,6 +47,7 @@ ksp = "2.0.20-1.0.25"
 chimp-jsocks-lib = { strictly = '567e1cd6' }
 failsafe-lib = { strictly = '3.2.4' }
 
+qrKit = "3.0.5"
```

Comment:
please remove qr dependency as we decided this will go in a separate PR

Reply:
Done